sdk/eventgrid/eventgrid/README.md

```diff
@@ -212,16 +212,23 @@ const client = new EventGridPublisherClient(
   new AzureKeyCredential("<API key>")
 );
 
-await client.send([
-  {
-    eventType: "Azure.Sdk.SampleEvent",
-    subject: "Event Subject",
-    dataVersion: "1.0",
-    data: {
-      hello: "world",
-    },
-  },
-]);
+async function main(){
```

Comment:
do we need to do this for snippets? I thought it was implicit that you would tend to put this code into a method body.

Also, I believe top level await is supported as of Node 14.8

Reply:
Await is only supported in async function. So we give a suggestion that we put the code into an async function. We think this modification can make it more convenient for users to use this code.

Reply:
This is no longer true in recent Node versions:

<img width="298" alt="image" src="https://github.com/Azure/azure-sdk-for-js/assets/639216/9ab2be7e-6ee9-42da-9e87-78c3a79db32a">

Reply:
Ok, it is not a question now. I will the close the PR and the related issue.